src/Http/Controllers/CP/Utilities/UpdateSearchController.php

```diff
@@ -17,10 +18,15 @@ public function update(Request $request)
 
         $indexes->each(function ($index) {
             [$name, $locale] = explode('::', $index);
+
             if ($locale) {
                 $name = Str::before($name, '_'.$locale);
             }
-            Search::index($name, $locale ?: null)->update();
+
+            $index = Search::index($name, $locale ?: null);
+            $index->update();
+
+            SearchIndexUpdated::dispatch($index);
```

Comment:
Move event into `$index->update()`.

Reply:
Done!

I didn't do it this way originally since there's no "central" `update` method for indexes, so indivdiual indexes will have to dispatch it.

Reply:
Oh, yeah. Hmm. 🤔 I think the way you had them was more correct.

Reply:
Changed it back!